cmd/tenant/main.go

```diff
@@ -29,6 +37,33 @@ func main() {
 			Action:   serve,
 			Flags:    []cli.Flag{},
 		},
+		{
+			Name:     "db:list",
+			Usage:    "list all keys in a tenant namespace",
+			Category: "client",
+			Action:   listKeys,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "namespace",
+					Aliases:  []string{"n"},
+					Usage:    "namespace to list keys for",
+					Required: true,
+				},
+			},
+		},
+		{
+			Name:     "db:migrate:orgs",
```

Comment:
```suggestion
			Name:     "db:reindex",
```

Re-indexing may be something that we need to do periodically across all of our indexes; so I suggest we merge this as a general purpose utility and identify other indexes that could be updated (completely) to fix problems.

Reply:
Good idea!